src/java.xml/share/classes/module-info.java

```diff
@@ -149,10 +149,10 @@
  * or reading the file, the configuration process proceeds as if the file does not exist.
  *
  * <h4 id="Conf_CF_SP">User-defined Configuration File</h4>
- * The system property {@systemProperty java.xml.config.file} can be set on the
- * command line or at run-time to specify the location of a configuration file.
- * If the {@code java.xml.config.file} property is included within a configuration
- * file, it will be ignored.
+ * In addition to the {@code jaxp.properties} file, the system property
+ * {@systemProperty java.xml.config.file} can be set on the command line or at run-time
```

Comment:
I think we can simplify to  "...the system property  {@systemProperty java.xml.config.file} can be set to specify..."

Reply:
Thanks Lance. Adjusted the javadoc accordingly.